packages/interface/interface.json

```diff
@@ -1630,12 +1630,6 @@
                 "setNotLoading": {
                   "desc": "removes loading indicator"
                 },
-                "setMaxHeight": {
```

Comment:
It's not necessary to manually override the max height of the notifications' flyout.